### PR TITLE
a fix for the build_cluster_test. adds the yaml. works with stopped services.

### PR DESCRIPTION
**Pull request by harelassayag**
this PR is instead of #422

### CODE REVIEW

**Comment by roydahan** [comment]
Overall looks ok.
Did you test the final version?

**Comment by harelassayag** [comment]
yes. works.

File: tests/build-cluster-dns.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            ami_monitor_user: 'centos'
+        stop_services: 'true'
+
+databases: !mux
+    cassandra:
+        db_type: cassandra
+        instance_type_db: 'm3.large'
+    scylla:
+        db_type: scylla
+        instance_type_db: 'i2.4xlarge'
```
Comment (roydahan) [inline tests/build-cluster-dns.yaml]: Please change the type to i3.2xlarge.

File: tests/build-cluster-dns.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+test_duration: 5760
+n_db_nodes: 3
+n_loaders: 1 
+n_monitor_nodes: 1
+user_prefix: 'build-cluster-dns'
```
Comment (roydahan) [inline tests/build-cluster-dns.yaml]: Please change the prefix to: cases-dns-VERSION.
The jenkins job should replace (sed) the VERSION with the correct version.

File: tests/build-cluster-dns.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+failure_post_behavior: destory
+ip_ssh_connections: 'private'
+experimental: 'true'
+
+
+backends: !mux
+    aws: !mux
+        cluster_backend: 'aws'
+        instance_type_loader: 'c4.4xlarge'
+        instance_type_monitor: 'c4.2xlarge'
```
Comment (roydahan) [inline tests/build-cluster-dns.yaml]: please change monitor to t2.small.

File: build_cluster_test.py [hunk 4]
```diff
@@ -66,24 +66,38 @@ def test_use_public_dns_names(self):
...
+                node.wait_jmx_up()
+
+        for node in self.db_cluster.nodes:
+            if not node.is_seed:
+                node.remoter.run('sudo systemctl start scylla-server.service')
+                node.remoter.run('sudo systemctl start scylla-jmx.service')
+                node.wait_db_up(timeout=300)
+                node.wait_jmx_up()
 
         base_cmd_w = ("cassandra-stress write no-warmup cl=QUORUM duration=5m "
```
Comment (roydahan) [inline build_cluster_test.py]: Take out the stress-cmd from the code and include it in the YAML.

Comment (roydahan) [reply build_cluster_test.py]: Also increase the duration to 15m.

File: tests/build-cluster-dns.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
+test_duration: 5760
```
Comment (roydahan) [inline tests/build-cluster-dns.yaml]: 1. Change the test duration to 15m.
2. Change the name of the YAML to dns-cluster-15min.yaml

File: tests/build-cluster-dns.yaml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+            subnet_id: 'subnet-d934e980'
+            ami_id_db_scylla: 'ami-e79e069d'
+            ami_db_scylla_user: 'centos'
+            ami_id_loader: 'ami-e79e069d'
+            ami_loader_user: 'centos'
+            ami_id_db_cassandra: 'ami-ada2b6c4'
+            ami_db_cassandra_user: 'ubuntu'
+            ami_id_monitor: 'ami-e79e069d'
+            ami_monitor_user: 'centos'
+        stop_services: 'true'
```
Comment (roydahan) [inline tests/build-cluster-dns.yaml]: Move this param above the mux (after experimental).

File: build_cluster_test.py (hunk 4 above)
Comment (roydahan) [reply build_cluster_test.py]: Add verify_stress_thread after the stress is done.

**Comment by roydahan** [comment]
@amoskong, please backport this to 2.0 and 2.1.
It shouldn't affect other tests.

**Comment by amoskong** [comment]
done.

**Pull request merged**